psi4/driver/p4util/util.py

```diff
@@ -138,13 +138,12 @@ def set_memory(inputval):
         units = ''
     # Handle memory given as a string
     if isinstance(inputval, basestring):
```

Comment:
Could also check `basestring` as in 
```
try:
    basestring
except NameError:
    basestring = str
```
which would then work with the current checks